assets/js/editor.js

```diff
@@ -342,12 +344,26 @@ $gp.editor = (
 				} );
 
 				if ( translationChanged ) {
-					$gp.notices.error( 'Translation has changed! Please add the new translation before changing its status.' );
+					$gp.notices.error( wp.i18n.__( 'Translation has changed! Please add the new translation before changing its status.', 'glotpress' ) );
 					return;
 				}
 
 				button.prop( 'disabled', true );
-				$gp.notices.notice( 'Setting status to &#8220;' + status + '&#8221;&hellip;' );
+
+				switch ( status ) {
+					case 'current':
+						status_name = wp.i18n._x( 'current', 'Single Status', 'glotpress' );
+						break;
+					case 'rejected':
+						status_name = wp.i18n._x( 'rejected', 'Single Status', 'glotpress' );
+						break;
+					case 'fuzzy':
+						status_name = wp.i18n._x( 'fuzzy', 'Single Status', 'glotpress' );
+						break;
+				}
```

Comment:
I think we need to add the "changes requested" status here:
```suggestion
					case 'changesrequested':
						status_name = wp.i18n._x( 'changes requested', 'Single Status', 'glotpress' );
						break;
				}
```

Reply:
Yes, you are right! I have added the .json files with the `wp i18n make-json glotpress-gl_ES.po` command in the `wp-content/languages/plugins` and the PR works fine!

Reply:
Indeed, will commit it right away.